<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graficar</class>
 <widget class="QDialog" name="Graficar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="VistaGrafo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>881</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValorVer">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelValor">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAgregarVer">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Vertice</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCosto">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Costo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditCosto">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditOrigen">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelOrigen">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Origen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDestino">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelDestino">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Destino:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAgregarArista">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>110</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Arista</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditDestinoBo">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnBorrarArista">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>350</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar Arista</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelOrigenBo">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Origen:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnBorrarVertice">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>320</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar Vertice</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditOrigenBo">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>360</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelDestinoBo">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>390</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Destino:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelVerticeBo">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertice:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditVerticeBo">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelTipo">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>240</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CmbOrigen">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>240</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCrearGrafo">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Grafo</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelEliminarGrafo">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Grafo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
